apps/stripe/src/app/api/saleor/transaction-charge-requested/use-case-response.ts

```diff
@@ -0,0 +1,75 @@
+import { buildSyncWebhookResponsePayload } from "@saleor/app-sdk/handlers/shared";
+
+import { SaleorMoney } from "@/modules/saleor/saleor-money";
+import { SuccessWebhookResponse } from "@/modules/saleor/saleor-webhook-responses";
+import { StripeCapturePaymentIntentAPIError } from "@/modules/stripe/stripe-payment-intent-api-error";
+import { StripePaymentIntentId } from "@/modules/stripe/stripe-payment-intent-id";
+
+class ChargeSuccess extends SuccessWebhookResponse {
+  readonly result = "CHARGE_SUCCESS";
+  readonly actions = []; // TODO: figure out what actions are available here
```

Comment:
refund

Reply:
Makes sense - I will add it in PR that adds support for refund globaly